profiles/base.yaml

```diff
@@ -255,6 +256,7 @@ classes_to_teach:
 avoid_athletics_in_justice: false
 athletics_outdoorsmanship_rooms:
 attunement_rooms:
+pray_to_chadatru: false
```

Comment:
Could this be renamed to something more specific? The name alone doesn't tell me if this is _all_ prayer or just some.

Reply:
I am not sure what you mean by all or some. Any suggestions?